source/validate.cpp

```diff
@@ -286,6 +291,12 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInsts,
 }
 
 namespace {
+// Shame
```

Comment:
That seems dubious.  How does `spvCheckReturn` work, then?  And all the places in the code where we already do exactly what I wrote above.